### PR TITLE
Minor changes to improve quality

### PR DESCRIPTION
**Pull request by bharathcs**

### CODE REVIEW

File: src/main/java/donnafin/ui/UiManager.java [hunk 1]
```diff
@@ -183,10 +178,8 @@ public ViewFinderState getUiState() {
     public CommandResult execute(String commandText) throws CommandException, ParseException {
         CommandResult commandResult = logic.execute(commandText);
         Consumer<Ui> uiAction = commandResult.getUiAction();
-        assert !uiAction.equals(null) : "commandResult.uiAction was set as null";
-        if (!uiAction.equals(null)) {
-            uiAction.accept(this);
-        }
+        assert uiAction != null : "commandResult.uiAction was set as null";
```
Comment (Bluntsord) [inline src/main/java/donnafin/ui/UiManager.java:179]: Theres 1 more in logic manager. I refactor that later?

**Review by Bluntsord — DISMISSED** [review]
Looks good :)

File: src/main/java/donnafin/ui/UiManager.java (hunk 1 above)
Comment (bharathcs) [reply src/main/java/donnafin/ui/UiManager.java:179]: sure

File: src/test/java/donnafin/commons/core/types/MoneyTest.java [hunk 2]
```diff
@@ -10,49 +10,50 @@
 public class MoneyTest {
```
Comment (Bluntsord) [inline src/test/java/donnafin/commons/core/types/MoneyTest.java:11]: Do u wanna comment somewhere here about the appropriate partitions and which test addresses whcih

File: src/main/java/donnafin/ui/ClientPanel.java [hunk 3]
```diff
@@ -141,6 +133,7 @@ private void changeTab(Node node) {
             refresh();
             attributeDisplayContainer.getChildren().add(node);
         });
-        attributeDisplayContainer.setVgrow(node, Priority.ALWAYS);
+        VBox.setVgrow(node, Priority.ALWAYS);
```
Comment (Bluntsord) [inline src/main/java/donnafin/ui/ClientPanel.java:136]: How does this work. Arent we specifyign attributeDisplayContainer to be the one that grows

File: src/test/java/donnafin/commons/core/types/MoneyTest.java (hunk 2 above)
Comment (bharathcs) [reply src/test/java/donnafin/commons/core/types/MoneyTest.java:11]: i don't have to, my functions are named clearly enough tho?

File: src/main/java/donnafin/ui/ClientPanel.java (hunk 3 above)
Comment (bharathcs) [reply src/main/java/donnafin/ui/ClientPanel.java:136]: nope, we are not, this line actually sets node to be always growing. [`setVgrow`](https://docs.oracle.com/javase/8/javafx/api/javafx/scene/layout/VBox.html#setVgrow-javafx.scene.Node-javafx.scene.layout.Priority-) is a static method, it never relied on `attributeDisplayContainer`

File: src/test/java/donnafin/commons/core/types/MoneyTest.java [hunk 4]
```diff
@@ -10,49 +10,55 @@
...
 
     @Test
-    public void getMoneyFromIntegerCheckValue() throws MoneyException {
+    public void constructor_positiveValue_success() throws MoneyException {
         assertEquals(123, new Money(123).getValue());
     }
-
+
```
Comment (bharathcs) [inline src/test/java/donnafin/commons/core/types/MoneyTest.java:16]:
```suggestion

```

File: src/test/java/donnafin/commons/core/types/MoneyTest.java [hunk 5]
```diff
@@ -10,49 +10,54 @@
 public class MoneyTest {
 
     @Test
-    public void getMoneyFromIntegerCheckValue() throws MoneyException {
+    public void constructor_positiveValue_success() throws MoneyException {
         assertEquals(123, new Money(123).getValue());
     }
-
     @Test
```
Comment (bharathcs) [inline src/test/java/donnafin/commons/core/types/MoneyTest.java:16]:
```suggestion

    @Test
```

File: src/main/java/donnafin/ui/ClientPanel.java (hunk 3 above)
Comment (Bluntsord) [reply src/main/java/donnafin/ui/ClientPanel.java:136]: aight. Understand. Went to test it out as well

**Review by Bluntsord — APPROVED** [review]
Aight looks nice! Thanks so much

**Pull request merged**